kcidb/monitor/subscriptions/linux_stable_rc.py

```diff
@@ -0,0 +1,35 @@
+"""Linux stable-rc subscription"""
+from datetime import (timezone, datetime, timedelta)
+
+from kcidb.monitor.output import NotificationMessage as Message
+
+
+def match_revision(revision):
+    """Match revisions of stable-rc tree"""
+    # Repo we're interested in
+    repo_url = "https://git.kernel.org/pub/scm/linux/kernel/git/stable/\
+linux-stable-rc.git"
```

Comment:
Please wrap this like this instead (with clearer indentation):
```python
    repo_url = "https://git.kernel.org/pub/scm/linux/kernel/git/stable/" \
        "linux-stable-rc.git"
```

Reply:
Fixed it.